### ISSUE TITLE
Ajouter "sélectionnez" au lieu d'un choix vide pour choix simple/multiple

### ISSUE THREAD

**Issue by lisa-durand** [demarches-simplifiees/demarches-simplifiees.fr#10188]
![Image](https://github.com/demarches-simplifiees/demarches-simplifiees.fr/assets/6756627/896dc0f8-67e8-45f3-beff-74a01870d5dc)

**Comment by colinux** [comment]
La doc du dsfr mentionne qu'on doit écrire "Sélectionner une option"

**Comment by mfo** [comment]
@marleneklok  – on utilise un bout de code existant fait pour l'accessibilité (https://react-spectrum.adobe.com/react-spectrum/ComboBox.html), malheureusement celui ci ne supporte pas ce type d'option. Jouable d'utiliser le hint a la place

ex sur un choix simple
![Image](https://github.com/user-attachments/assets/faade323-7a71-421a-bd49-24fc7412ca05)

ex sur un choix multiple
![Image](https://github.com/user-attachments/assets/0fda0951-c1c0-4344-96fc-70fcc4dbef9d)

une autre option serait d'utiliser un placeholder, mais pas sur que ca passe côté accessibilité, cc @inseo  ex:
![Image](https://github.com/user-attachments/assets/742f602c-7442-4c80-bd23-29882a007755)

**Comment by marleneklok** [comment]
@mfo Je pense qu'il faut qu'on échange sur ces nouvelles "listes déroulantes" avec beaucoup de choix, qui ne sont plus des listes déroulantes au final, car elles exigent de commencer à saisir qqch pour avoir des suggestions. Ce comportement est ok pour le champ type "Adresse", par contre pour les listes déroulantes "Choix simple" ou "Choix multiple", il faut selon moi conserver le comportement de liste déroulante et permettre d'accéder plus rapidement aux choix "connus" via la saisie.
Donc l'icône à conserver est le chevron (pour liste déroulante avec possibilité de saisie) :
![image](https://github.com/user-attachments/assets/6b1f0ec1-8bab-409e-8054-f192efb447a5)
Ce qui est différent de l'icône "recherche" pour les champs qui exigent de la saisie pour proposer des suggestions :
![image](https://github.com/user-attachments/assets/11a59d16-3345-46d7-b99f-33ee39c84c16)
A dispo pour s'en parler de vive voix ;)

**Comment by marleneklok** [comment]
Recos selon les 2 comportements A et B des listes déroulantes (en fonction du type de champ) :
[USAGER_Liste déroulante.pdf](https://github.com/user-attachments/files/16846755/USAGER_Liste.deroulante.pdf)

**Comment by Benoit-MINT** [comment]
le DSFR projette ça : https://www.systeme-de-design.gouv.fr/version-courante/fr/composants/liste-deroulante-riche

**Comment by marleneklok** [comment]
@Benoit-MINT Oui, ou même plus simplement : https://www.systeme-de-design.gouv.fr/version-courante/fr/composants/liste-deroulante
L'idée serait juste d'ajouter "Sélectionner" (plutôt qu'une valeur vide) dans les listes déroulantes.
